tools/cocos2d-console

```diff
@@ -1 +1 @@
-Subproject commit 2ceba42b89945229ac22d09c0864c86b113f3f17
```

Comment:
Apparently it was fixed here: https://github.com/cocos2d/cocos2d-x/commit/3e33b7a1cad14dab8bd896c270e319e2d12a78e3
thanks.